packages/cli/src/parse_async_safely.test.ts

```diff
@@ -11,11 +10,7 @@ const mockParser: Argv = {
 
 void describe('execute parseAsyncSafely', () => {
   void it('parseAsyncSafely should not throw an error', async () => {
-    try {
-      await parseAsyncSafely(mockParser);
-      assert.ok(true, 'parseAsyncSafely did not throw an error');
-    } catch (err) {
-      assert.fail('parseAsyncSafely threw an error');
-    }
+    await parseAsyncSafely(mockParser);
+    //not throw
```

Comment:
```suggestion
    // no throw
```

or `// doesn't throw`